main/src/ca/uwaterloo/flix/api/lsp/provider/SemanticTokensProvider.scala

```diff
@@ -625,6 +625,21 @@ object SemanticTokensProvider {
   private def visitExps(exps0: List[Expr]): Iterator[SemanticToken] =
     exps0.flatMap(visitExp).iterator
 
+  private def visitCst(constant: Expr.Cst): Iterator[SemanticToken] = constant.cst match {
+    case _ : Constant.Str |
```

Comment:
Writing the pattern match this way is not idiomatic. Just match on the constructor and write Str(_, _).

Reply:
Oh, and also, we dont use | or patterns. They are fragile. Just write out the normal pattern match 👍